backend/src/src-licenseinfo/src/main/java/org/eclipse/sw360/licenseinfo/outputGenerators/DocxGenerator.java

```diff
@@ -78,6 +81,8 @@
     private static final String EXT_ID_TABLE_HEADER_COL1 = "Identifier Name";
     private static final String EXT_ID_TABLE_HEADER_COL2 = "Identifier Value";
 
+    private static final Map<LicenseNameWithText, Integer> licenseToReferenceId = Maps.newHashMap();
```

Comment:
Please use getter method or declare it inside a method instead of creating constant.

Reply:
Thanks for pointing it out. I have incorporated the changes.